malaria-ios/Managers/UserSettingsManager.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+class UserSettingsManager{
+    
+    /// Sets an UserSetting
+    /// :param: `UserSetting`: the setting to be changed
+    /// :param: `Bool`: value
+    class func setBool(key: UserSetting, _ value: Bool){
```

Comment:
Instead of these helper methods, you should try just implementing
`UserSettingsManager.didConfiguredMedicine` / `UserSettingsManager.setDidConfiguredMedicine` etc that way you can consolidate your string keys here as well.  Or extend `NSUserDefaults` so you can just make calls like `NSUserDefaults.standardUserDefaults().setDidConfigureMedicine` etc.  Same idea.  Let me know if that make sense.